content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -26,7 +26,7 @@ Endpoints.
 
 ## Endpoint Slice resources {#endpointslice-resource}
 
-In Kubernetes, an Endpoint Slice contains references to a set of network
+In Kubernetes, an EndpointSlice contains references to a set of network
 endpoints. The EndpointSlice controller automatically creates Endpoint Slices
```

Comment:
(if #16885 lands, I can add a page about that specific controller).

Reply:
Nice! I've got some slides that help explain the logic in that controller if that's helpful.